name, age = ('Name', 18)
# name, age, city = ('Name', 18) В таком случае будет ошибка
# name = user[0]
# age = user[1]
print(name)
print(age)

a,b = "cd"
print(a)
print(b)

print('_'*50)
a, b = (1, 2)
temp = (a, b)
b, a = temp
# temp1 = a
# temp2 = b
# a = temp2
# b = temp1
print(a,b)
# Присвоение у нас работает сначала происходит расчет правой части, потом происходит присвоение. 
# Сначала рассчтывает, что а = 1, и b = 2, а потом подставляет а - b, и 1 -> b и 2 -> a

x, y = (10+ 4*5, "aaa") #т.е. тут сначалп роисходит расчет правой части, а тольо потом присвоение
print(x, y)


print('_'*50)